docs/access-api-spec.md

```diff
@@ -410,6 +411,36 @@ rpc GetTransactionResult (GetTransactionRequest) returns (TransactionResultRespo
 
 ### Accounts
 
+#### GetAccount
+
+`GetAccount` gets an [account](#account) by address at the latest sealed block.
+
+Note: this function behaves identically to `GetAccountAtLatestBlock` and will be deprecated in a future version.
```

Comment:
isn't it already deprecated?

Reply:
Ah true, I'll update both comments to reflect this. Thanks!